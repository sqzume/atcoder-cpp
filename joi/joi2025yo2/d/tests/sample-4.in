13 12 10
2 28 28 60 48 77 63 92 13 71 36 91 87
85 7 64 15 55 92 66 91 83 35 49 22 61
2 9
8 13
7 11
9 11
8 12
5 12
4 7
11 12
10 12
4 11
1 5
3 8
49 21 46 13 20 41 6 33 24 7
