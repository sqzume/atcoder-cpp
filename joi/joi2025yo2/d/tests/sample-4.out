121
169
129
174
169
137
183
148
169
183
